src/sentry/api/endpoints/project_details.py

```diff
@@ -814,3 +833,12 @@ def _fix_rule_ids(self, project, raw_dynamic_sampling):
 
         raw_dynamic_sampling["next_id"] = next_id
         return raw_dynamic_sampling
+
+    def _dynamic_sampling_contains_error_rule(self, raw_dynamic_sampling):
+        if raw_dynamic_sampling is not None:
+            rules = raw_dynamic_sampling.get("rules", [])
+            for rule in rules:
+                if rule["type"] == "error":
+                    return True
+                else:
+                    return False
```

Comment:
Do you only want to check the first rule or all of them? This looks like it will only check the first rule. If you only need to check the first rule you likely don't need a loop.

Reply:
good catch! thanks